webapp/src/services/utils/index.ts

```diff
@@ -148,8 +148,14 @@ export const exportText = (fileData: string, filename: string): void => {
   link.remove();
 };
 
-export const displayVersionName = (version: string): string =>
-  version.split("").join(".");
+export const displayVersionName = (version: string): string => {
+  const [major, minor, patch] = version.split("");
+  // cf. https://github.com/AntaresSimulatorTeam/AntaREST/issues/1204
+  if (patch === "0") {
+    return `${major}.${minor}`;
+  }
+  return `${major}.${minor}.${patch}`;
+};
```

Comment:
you dont want to change this one has you will affect the study versions displayed on the entire application by removing the last digit.

instead define a new util function to format solver versions:

```javascript
const displaySolverVersions = (versions: string[]): string[] => 
  versions.map(v => `${v[0]}.${v[1]}`);
```
and keep the original one by renaming properly to: `displayStudyVersions` or `formattedStudyVersions` as you like
that will keep the same format for study versions:

```javascript
const displayStudyVersions = (versions: string[]): string[] => 
  versions.map(v => `${v[0]}.${v[1]}.${v[2]}`);
```

Reply:
> Edit: in fact we want to change all displayed versions and exclude the patch digit.

```javascript
const displayVersion = (v: string): string => `${v[0]}.${v[1]}`;
```